src/app/header/header.component.ts

```diff
@@ -6,6 +6,14 @@ import { StoreService } from '../store.service';
 @Component({
   selector: 'Header',
   template: `
+    
+    <!-- Този back btn се появява само във формата, но това е отвъд моите способности :) -->
```

Comment:
We're open to pull requests from all nationalities, let's use English everywhere except for the i18n object definitions :)